tests/requirements/integration/deb-requirements.txt

```diff
@@ -0,0 +1,3 @@
+docker
+docker-compose
+python-pip
```

Comment:
Should this be either `python2-pip` or `python3-pip`? Or both?

Reply:
The package is actually `python-pip`, the third version, as you spotted, has the `3`. It has not errored when running in the VM still though, so where is pyyaml actually needed? I guess it is for the release-tool?

Reply:
Yes, it is for release_tool